### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by Jiawei-Yang** [FoundationVision/VAR#22]
Hi there,

thanks for your amazing work! I noticed the recent updates covered the training code. Will there also be updates about training scripts and data preparation? I'm particularly interested in training the VQVAE part.

**Comment by iFighting** [comment]
> Jiawei-Yang

@Jiawei-Yang thanks for your kindly attention, we will release the training code of VQVAE in several days

**Comment by shliu0** [comment]
> > Jiawei-Yang
> 
> @Jiawei-Yang thanks for your kindly attention, we will release the training code of VQVAE in several days

I am interested in replicating your vqvae results and would appreciate some additional details to help me understand and possibly extend your work.  Could you please provide more specifics regarding the training configuration? 
e.g. exact loss function equation used during training and weights for each term, batch size, lr, etc.
Thanks for your good work!

**Comment by pkulwj1994** [comment]
Also very interested in the training of VQ-tokenizer part, looking forward! 
BTW: VAR is a super cool work!
